asm/c64/axpyincto_amd64.s

```diff
@@ -0,0 +1,52 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//+build !noasm,!appengine
+
+#include "textflag.h"
+
+//func AxpyIncTo(dst []complex64, incDst, idst uintptr, alpha complex64, x, y []complex64, n, incX, incY, ix, iy uintptr)
+TEXT ·AxpyIncTo(SB), NOSPLIT, $0
+	MOVQ	dst_base+0(FP), R8
+	MOVQ	x_base+48(FP), SI
+	MOVQ	y_base+72(FP), DI
+	MOVQ	n+96(FP), DX
+	XORQ	AX, AX		// i = 0
+	CMPQ	DX, $0		// if n==0, return
+	JE	axpyi_end
+	MOVQ	ix+120(FP), CX		//Load the first indicies
+	MOVQ	iy+128(FP), BX
+	MOVQ	idst+32(FP), R9
+	LEAQ	(SI)(CX*8), SI		//Calculate addrress of first indicies
```

Comment:
Sorry, spelling/grammar not semantics; too many 'r's in address and indicies is plural.